.eslintrc

```
@@ -0,0 +1,49 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2020
+  },
+  "extends": ["eslint:recommended", "prettier"],
+  "ignorePatterns": ["node_modules/", "dist/", "work/", "coverage/"],
+  "overrides": [
+    {
+      "files": ["*.{ts,tsx}"],
+      "extends": [
+        "plugin:@typescript-eslint/eslint-recommended",
+        "plugin:@typescript-eslint/recommended",
+        "prettier/@typescript-eslint"
+      ],
+      "rules": {
+        "@typescript-eslint/explicit-function-return-type": "off",
+        "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/no-inferrable-types": "off",
+        "@typescript-eslint/no-non-null-assertion": "off",
+        "@typescript-eslint/no-require-imports": "off",
+        "@typescript-eslint/no-unused-vars": "off",
+        "@typescript-eslint/no-use-before-define": "off",
+        "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/camelcase": "off"
+      }
+    },
+    {
+      "files": [
+        "jest.*.js",
+        "test/**/*.{ts,tsx,js,jsx}",
+        "*.test.{ts,tsx,js,jsx}",
+        "__mocks__/**/*.{ts,tsx,js,jsx}"
+      ],
+      "rules": {
+        "@typescript-eslint/no-non-null-assertion": "off"
+      }
+    }
+  ],
+  "rules": {
+    "no-unused-vars": "off",
+    "no-undef": "error"
+  },
+  "env": {
+    "node": true,
+    "es6": true
+  }
+}
```

Updating to the latest SDK libraries and adopting the latest versions of files in https://github.com/JupiterOne/integration-template would be a worthwhile inclusion in this initial release. For example, the new version of this particular file becomes:
```suggestion
{
  "root": true,
  "extends": [
    "./node_modules/@jupiterone/integration-sdk-dev-tools/config/eslint.json"
  ]
}
```

---

Sounds good,

I used the command you gave from snipe-it to pull over most files, but it looks like this one was not included.
```
cp ~/Workspaces/j1/gh/integration-template/{husky.config.js,jest.config.js,jest.env.js,lint-staged.config.js,prettier.config.js,tsconfig.dist.json,tsconfig.json} ./
```
I'll be sure to update the above command to include `eslint.rc` as well.

---

Looking at this again, I'll need to update a lot of other files as well to get this project in sync with the integration template. I'll take this opportunity to update the entire project. 

- [ ] Update project to be in line with the integration template.